setup.py

```diff
@@ -94,7 +94,10 @@
             "docs": doc_requirements,
             "test": test_requirements,
             "interactive": interactive_requirements,
-            "dev": doc_requirements + test_requirements + interactive_requirements,
+            "dev": doc_requirements
+            + test_requirements
+            + interactive_requirements
+            + ["black==22.3.0", "isort"],
```

Comment:
We don't want to declare dependencies here. We haven't settled on a formal model for including linting requirements to our framework repos, but I suspect we may settle on what was done [here](https://github.com/ihmeuw/vivarium_gates_nutrition_optimization_child/blob/main/setup.py#L37C35-L38C43)

Reply:
Sure, I can change it to that.

Reply:
@rmudambi I've updated this.